WilliamsGarcia/02-09/vetCrud/src/main/java/com/example/vetcrud/entity/Pet.java

```diff
@@ -0,0 +1,101 @@
+package com.example.vetcrud.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Column;
+import javax.persistence.ManyToOne;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+
+import java.io.Serializable;
+
+@Entity
+@Table(name = "pet_table")
+public class Pet implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "age", nullable = false)
+    private Integer age;
+
+    @Column(name = "specie", nullable = false)
+    private String specie;
+
+    @Column(name = "race", nullable = false)
+    private String race;
+
+    @Column(name = "isVaccinated", nullable = false)
```

Comment:
Use lower letters when naming columns. For separating words, use underscore https://www.sqlshack.com/learn-sql-naming-conventions/